test/functional/test-error.js

```diff
@@ -409,3 +410,40 @@ describes.sandboxed('reportError', {}, () => {
     }).to.throw(/_reported_ Error reported incorrectly/);
   });
 });
+
+describe.only('detectJsEngineFromStack', () => {
```

Comment:
Can you update the integration test glob to include this (so the tests actually run on SC)?